src/main/webapp/i18n/de/messages.json

```diff
@@ -0,0 +1,20 @@
+{
+    "artemisApp": {
+        "messages": {
+            "label": "Nachrichten",
+            "conversation": {
+                "header": "Gespräch",
+                "created": "Gespräch erstellt",
+                "show": "Gespräch anzeigen",
+                "hide": "Gespräch ausblenden",
+                "notFound": "Kein Gespräch gefunden",
+                "searchForUsers": "Suchen Sie Benutzer innerhalb des Kurses"
+            },
+            "conversationParticipant": {
+                "created": "Gesprächsteilnehmer erstellt",
```

Comment:
I cannot find any usage of this translation (?). However, from the end-user perspective you would **add** users to a conversation instead of **creating** users. Do you know what I mean?

Reply:
Stephan asked me to remove the usages. I got what you are pointing with *creating*. Thanks!